spec/javascripts/metrics_spec.js

```diff
@@ -104,38 +104,38 @@ describe('metrics.js', function() {
       { time: '2014-11-30T00:11:09.000+00:00', value: 123 },
       { time: '2014-12-01T00:10:56.000+00:00', value: 456 },
     ]
-
+  
     expect(getPoints(metrics, { prefix: '£', suffix: ''})).toEqual({
       x: [
-        '2014-11-30T00:11:09.000+00:00',
-        '2014-12-01T00:10:56.000+00:00',
+        '2014-11-30 00:11:09',
+        '2014-12-01 00:10:56',
       ],
       y: [
         123,
         456
-      ]
+      ],
       text: [
         '£123',
         '£456'
       ]
     })
   })
-
+  
   it ('adds a suffix to some text', function() {
     var metrics = [
       { time: '2014-11-30T00:11:09.000+00:00', value: 123 },
       { time: '2014-12-01T00:10:56.000+00:00', value: 456 },
     ]
-
+  
     expect(getPoints(metrics, { prefix: '', suffix: '%'})).toEqual({
       x: [
-        '2014-11-30T00:11:09.000+00:00',
-        '2014-12-01T00:10:56.000+00:00',
+        '2014-11-30 00:11:09',
+        '2014-12-01 00:10:56',
```

Comment:
for this and preceding fix was it the trailing zeroes or the T which was causing the problem? Asking as I'm wondering does this have ramifications for how the API is documented and what ISO-date-time values Bothan accepts.